spec/models/admin_set_spec.rb

```diff
@@ -98,16 +98,11 @@
   end
 
   it "has a title" do
-    subject.title = ["title"]
-    subject.save
-    expect(subject.reload.title).to eq ["title"]
+    expect(subject.title).to be_a(ActiveTriples::Relation)
```

Comment:
The changes to this spec are the only ones I have questions about in this PR. This test change gives me slightly less confidence that what I expect is actually happening. Can you explain this change?